packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsListEmptyStateCard.tsx

```diff
@@ -47,10 +51,25 @@ export const SettingsAccountsListEmptyStateCard = ({
     isMicrosoftCalendarEnabledState,
   );
 
+  const isImapEnabled = useIsFeatureEnabled(FeatureFlagKey.IS_IMAP_ENABLED);
+
+  const navigate = useNavigateSettings();
+
   return (
     <Card>
       <StyledHeader>{label || t`No connected account`}</StyledHeader>
       <StyledBody>
+        {isImapEnabled && (
+          <Button
+            Icon={IconMail}
+            title={t`Connect with IMAP`}
+            variant="secondary"
+            // Apparently <Link> Is broken so we resort to onClick
```

Comment:
Button has a `to` property, did you try it?I don't think we need to use link here